### PR TITLE
Update to 0.9.3

### LINKED ISSUE #101: update node to align with latest rococo
Update to polkadot@0.9.2, cumulus@0.9.2

### PR DESCRIPTION
**Pull request by stanly-johnson**
Resolves #101 

- [X] Update Parachain
- [X] Update standalone
- [x] Parachain builds block in local test

### CODE REVIEW

**Comment by stanly-johnson** [comment]
@olanod is the standalone node runtime needed anymore? 🤔

**Comment by olanod** [comment]
Might still be useful for development? unless is too cumbersome to keep both in sync 🤔

**Review by olanod — COMMENTED** [review]
I guess most of the changes to the node client are from upstream? If this works let's just merge it and then we continue figuring out how to bring back the inter-chain functionality

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -11,57 +11,58 @@ version = '2.0.0'
...
-cumulus-primitives-utility = {git = "https://github.com/paritytech/cumulus", default-features = false, branch = "rococo-v1"}
-cumulus-pallet-xcm = {git = "https://github.com/paritytech/cumulus", default-features = false, branch = "rococo-v1"}
-cumulus-pallet-xcmp-queue = {git = "https://github.com/paritytech/cumulus", default-features = false, branch = "rococo-v1"}
-pallet-xcm = {git = "https://github.com/paritytech/polkadot", default-features = false, branch = "rococo-v1"}
+cumulus-pallet-parachain-system = {git = "https://github.com/paritytech/cumulus", default-features = false, branch = "polkadot-v0.9.3"}
+cumulus-primitives-core = {git = "https://github.com/paritytech/cumulus", default-features = false, branch = "polkadot-v0.9.3"}
+parachain-info = {git = "https://github.com/paritytech/cumulus", default-features = false, branch = "polkadot-v0.9.3"}
+cumulus-pallet-aura-ext = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.3", default-features = false }
+# xcm = {git = "https://github.com/paritytech/polkadot", default-features = false, branch = "polkadot-v0.9.3"}
```
[the remark this answers is not in the document]
Comment (stanly-johnson) [reply runtime/Cargo.toml:51]: For our use case mostly yes, will wait for oral-xcm to come back

**Review by olanod — APPROVED** [review]

**Pull request merged**